leptonai/cli/photon.py

```diff
@@ -598,7 +602,8 @@ def run(
 
         if include_workspace_token:
             console.print("Including the workspace token for the photon execution.")
-            _add_workspace_token_secret_var_if_not_existing(conn)
+            secret = list(secret)
+            _add_workspace_token_secret_var_if_not_existing(conn, secret)
```

Comment:
Directly add

if "LEPTON_WORKSPACE_TOKEN" not in secret:
        secret.append("LEPTON_WORKSPACE_TOKEN")

here?